### PR TITLE
first draft of the news letter

### PR DESCRIPTION
**Pull request by FlanFlanagan**
@skc222 @katyhuff @gonuke @bam241 @scopatz 

First draft of the newsletter. I'll be making edits in the near future to add somethings in. But I figured I'd let those that have things they would like to add be able to add them.

### CODE REVIEW

**Comment by FlanFlanagan** [comment]
PR against this branch I think?

**Comment by bam241** [comment]
I did not see it was a cyclus/cyclus.github.com branch, sorry

**Comment by gonuke** [comment]
Should we add a list of currently funded projects 1-2 sentence summaries of their respsective goals?

File: source/newsletters/pubs.bib [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    booktitle = {{Global 2017}},
+    author = {Bae, J.W. and K. Huff, K. and Flanagan, R. and Scopatz, A.},
+    month = {September},
+    year = {2017}
+},
+
+@inproceedings{mcgarry_2017,
+    address = {Indian Wells, USA},
+    title = {{State-Level Decision-Making In Cyclus to Assess Multilateral Enrichment.}},
+    booktitle = {{39Th ESARDA Symposion}},
```
Comment (bam241) [inline source/newsletters/pubs.bib]: this should be "INMM 58th Annual Meeting"

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**